orchestra/frontend/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
:)

Reply:
i believe this isn't commented, but part of the build script for typescript -- https://stackoverflow.com/questions/60628874/what-does-reference-types-react-scripts-mean-what-other-xml-like-sy